doc/help/commands.asciidoc

```diff
@@ -277,8 +278,7 @@ Set all settings back to their default.
 
 [[config-cycle]]
 === config-cycle
-Syntax: +:config-cycle [*--pattern* 'pattern'] [*--temp*] [*--print*]
-             'option' ['values' ['values' ...]]+
+Syntax: +:config-cycle [*--pattern* 'pattern'] [*--temp*] [*--print*] 'option' ['values' ['values' ...]]+
```

Comment:
I think we've talked about this before, but I don't get why your Python does something different here to mine :laughing: